target/linux/ipq40xx/base-files/etc/board.d/02_network

```diff
@@ -215,6 +216,10 @@ ipq40xx_setup_macs()
 		wan_mac=$(mtd_get_mac_ascii devinfo hw_mac_addr)
 		lan_mac=$(macaddr_add "$wan_mac" 1)
 		;;
+	linksys,whw03)
+		wan_mac=$(mmc_get_mac_ascii devinfo hw_mac_addr)
+		lan_mac="$wan_mac"
```

Comment:
flipy's original port did `lan_mac=$(macaddr_add "$wan_mac" 1)` here. Is there a reason you chose to have the same MAC address for WAN and LAN?

Reply:
Well, I just compared this to the stock firmware, it does indeed seem to use the same MAC address for WAN and LAN. A bit strange in my opinion.

Reply:
added section.